x-pack/plugins/maps/public/routing/state_syncing/app_sync.js

```diff
@@ -10,7 +10,7 @@ import { map } from 'rxjs/operators';
 import { getData } from '../../kibana_services';
 import { kbnUrlStateStorage } from '../maps_router';
 
-export function useAppStateSyncing(appStateManager) {
+export function startAppStateSyncing(appStateManager) {
```

Comment:
Changed name from `use` to `start` so react does not complain about this function being a hook.

Reply:
That was why it was complaining? Interesting. I think using start is better here anyway :+1: